src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.svelte

```diff
@@ -48,43 +60,67 @@
     document={description?.json}
     links={description?.links}
     imageSizeType="col-9"
+    {pageMetadataMap}
   />
 {/if}
 
 {#if childServiceEntries.length > 0 || childServiceGroups.length > 0}
   <h2>{serviceListName}</h2>
-{/if}
 
-{#if childServiceEntries.length > 0}
-  <Accordion multiselectable>
-    {#each childServiceEntries as item}
-      <AccordionItem title={item?.entryTitle} id={item.sys.id}>
-        <ContentfulRichText
-          document={item?.description?.json}
-          links={item?.description?.links}
-          blurhashes={item?.description?.blurhashes}
-        />
-      </AccordionItem>
-    {/each}
-  </Accordion>
-{/if}
+  {#if childServiceEntries.length > 0}
+    <Accordion multiselectable bordered>
+      {#each childServiceEntries as item}
+        <AccordionItem title={item?.entryTitle} id={item.sys.id}>
+          <ContentfulRichText
+            document={item?.description?.json}
+            links={item?.description?.links}
+            blurhashes={item?.description?.blurhashes}
+            {pageMetadataMap}
+          />
+          {#if item.serviceCtaCollection}
+            {#each item.serviceCtaCollection.items as ctaItem}
+              <span class="service-entry-CTA">
+                <ContentfulRichText
+                  document={ctaItem?.callToActionDestination?.json}
+                  links={ctaItem?.callToActionDestination?.links}
+                  {pageMetadataMap}
+                />
+              </span>
+            {/each}
+          {/if}
 
-{#if childServiceGroups.length > 0}
-  <ul class="service-group-list">
-    {#each childServiceGroups as item}
-      <Card class="service-group-card">
-        <h3 class="usa-card__heading" slot="header">{item.title}</h3>
-        <svelte:fragment slot="body">
-          {#if item.subheading}
-            {item.subheading}
+          <!-- .filter is to make sure there aren't any null 
+            contact items(caused by contacts in a draft state) -->
```

Comment:
this was caused by one of the service entries on `/about/contact/complaints`. there's a contact that's still in a draft state and the response for `items` was `[null]`